lib/optimize/InnerGraphPlugin.js

```diff
@@ -202,6 +213,27 @@ class InnerGraphPlugin {
 							const fn = statementWithTopLevelSymbol.get(statement);
 							if (fn) {
 								InnerGraph.setTopLevelSymbol(parser.state, fn);
+								if (
```

Comment:
Simlar for ClassExpression in `parser.hooks.declarator`.

Need a test with class expressions.